keras_cv/models/object_detection_3d/center_pillar.py

```diff
@@ -158,7 +158,12 @@ def __init__(
         self._prediction_decoder = prediction_decoder
         self._head_names = self._multiclass_head._head_names
 
-    def call(self, point_xyz, point_feature, point_mask, training=None):
+    def call(self, input_dict, training=None):
+        point_xyz, point_feature, point_mask = (
+            input_dict["point_xyz"],
+            input_dict["point_feature"],
```

Comment:
feel free to target in a follow up --- but should we revisit any of these dictionary keys?  I lack context on the problem domain unfortunately so no insight from me here.

Reply:
Yes, more API updates to come for these models + layers to clean this up. For now, I just wanted to get standardized on a dictionary.